README.md

```diff
@@ -7,3 +7,29 @@ permutation groups of degree < 4096.
 
 It has been formerly a part of the core GAP system, and has been
 converted to a separate GAP package in 2017 for the GAP 4.9 release.
+
+## Documentation
+
+Full information and documentation can be found in the manual, available
+as PDF `doc/manual.pdf` or as HTML `htm/chapters.htm`, or on the package
```

Comment:
The starting page for HTML format is `doc/chap0.html`. The path that you've specified is for gapmacro-based manuals.